src/jsonserializer/jsonserializer.qbs

```diff
@@ -1,12 +1,30 @@
 import qbs
 import qbs.TextFile
 
-DynamicLibrary {
+Product {
     id: root
+
+    property bool isAndroid: qbs.targetOS.contains("android")
+    property bool isMacOS: qbs.targetOS.contains("macos")
+    property bool isWindows: qbs.targetOS.contains("windows")
```

Comment:
I don't think we should expose these properties.
Changing them externally is clearly not intended, nor necessary.

Reply:
I would suggest to inline these expressions into the conditions below.

Reply:
or add local variables with these names.